src/resources/filters/layout/table.lua

```diff
@@ -7,10 +7,15 @@ function tablePanel(divEl, layout, caption, options)
   if not options then
     options = {}
   end
-  
+  -- outer panel to contain css and figure panel
```

Comment:
@cscheid @jjallaire

@dragonstyle and I paired on this one.  Can you please review this fix to adding the id here.  We are curious if it was deliberately left out for a particular reason.  Thank you.

Reply:
Likely just an oversight, this LGTM